frontend/shared-ui/assets/styles/main.scss

```
@@ -238,8 +238,13 @@
 }
 
 .message-bubble {
-  @apply flex flex-col px-4 pt-2 pb-3 w-fit min-w-[30%] max-w-full border overflow-x-auto rounded-xl shadow-sm;
-  overflow-wrap: break-all;
+  // No min-width on phones: a 30% floor on a 390px viewport forces short
+  // messages wider than their content while leaving long ones nothing to gain.
+  @apply flex flex-col px-4 pt-2 pb-3 w-fit md:min-w-[30%] max-w-full border overflow-x-auto rounded-xl shadow-sm;
+  // break-word, not break-all: break-all splits ordinary words mid-character,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)